symplyphysics/laws/electricity/electrostatic_potential_is_work_to_bring_from_infinity_over_charge.py

```diff
@@ -0,0 +1,64 @@
+"""
+Electrostatic potential is work to bring from infinity over charge
+==================================================================
+
+Electrostatic potential is a physical quantity defined as the amount of
+work needed per unit electric charge to move it from a reference point,
+usually infinity, to a specific point in an electric field.
+
+Infinity is usually used as the reference point because this would make
+the potential approach zero at an infinitely remote point.
+"""
+
+from sympy import (Eq, solve)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    validate_input,
+    validate_output,
+)
+
+electrostatic_potential = Symbol("electrostatic_potential", units.voltage)
+"""
+Electrostatic potential of a point in an electric field.
+
+Symbol:
+    :code:`V`
+"""
+
+work = Symbol("work", units.energy)
+"""
+Work needed to bring the charge from the infinity.
```

Comment:
This is more of a theoretical definition than a practical one; moreover one might view infinity as a limit (cf. thermodynamic limit where the number of particles approaches infinity as a way to model an infinite number of particles), but I agree we can call it a "reference point" since potential is defined up to a constant.